src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -684,7 +695,7 @@ private void AddServicesProducedInfo(IResource modelResource, IAnnotationHolder
             }
             catch (Exception ex)
             {
-                Console.WriteLine($"Could not stop {resourceName} '{res.Metadata.Name}': {ex}");
+                _logger.LogInformation(ex, "Could not stop {resourceName} '{res.Metadata.Name}'.", resourceName, res.Metadata.Name);
```

Comment:
What is going on with the names in this log message? `res.Metadata.Name` in the string should the logical name, not an internal C# indentifier. Also, we tend to use pascal case for structured logging names.

Reply:
My bad. I cut and paste from the `Console.WriteLine(...)` that was there previously which had the interpolated string. Tweaks incoming. Can't see a lot of evidence of preferencing Pascal Case though? Might not be looking in the right places.

Reply:
https://github.com/search?q=repo%3Adotnet%2Faspnetcore%20LoggerMessage&type=code

Reply:
I stand corrected!

Reply:
Went through and fixed up the others I found.